src/components/Navbar.tsx

```diff
@@ -1,4 +1,13 @@
+import cn from 'classnames';
+import { NavLink, useLocation } from 'react-router-dom';
+
 export const Navbar = () => {
+  const getIsActive = ({ isActive }: { isActive: boolean }) =>
```

Comment:
import it from utils and give it better name or remove getIsActive

Reply:
Good point